<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QQ</string>
  </property>
  <widget class="QLineEdit" name="leipAddress">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leport">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>8082</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textReceive">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>381</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>tips:点击头像、昵称等可进行设置。</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>点击下载路径进行选择，或手动输入。</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>双击文文件进行下载。</string>
    </property>
   </item>
  </widget>
  <widget class="QPlainTextEdit" name="textSend">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>IP地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>端口号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectBtn_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QPushButton" name="image_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>头像：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>昵称：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameEd">
      <property name="text">
       <string>小明</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送文件</string>
   </property>
  </widget>
  <widget class="QListView" name="view">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>200</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>380</y>
     <width>216</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>下载路径</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>C://Users//davincideng//Desktop//1//file</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
